src/components/Tokens/TokenTable/TimeSelector.tsx

```diff
@@ -52,7 +54,7 @@ const InternalLinkMenuItem = styled(InternalMenuItem)`
     text-decoration: none;
   }
 `
-const MenuTimeFlyout = styled.span`
+const MenuTimeFlyout = styled.span<{ isExplore: boolean }>`
```

Comment:
we might want to right align or shrink the row width, this menu is cut off on medium screen widths
<img width="809" alt="Screenshot 2023-10-02 at 5 23 14 PM" src="https://github.com/Uniswap/interface/assets/11512321/5f2ab09a-2294-429d-ac52-1045683c86e4">